Remove "cute" configure message that's misformatted due to non-POSIX use
of echo -n.
--- configure.orig	2003-01-14 03:39:59.000000000 -0600
+++ configure	2020-01-28 04:55:59.000000000 -0600
@@ -1876,9 +1876,6 @@
 
 
 
-echo -n "checking for feather on bird..."
-sleep 1
-echo " yes"
 
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
